docs/source/deployment/airflow_astronomer.md

```diff
@@ -2,161 +2,153 @@
 
 Apache Airflow is a popular open-source workflow management platform. It is a suitable engine to orchestrate and execute a pipeline authored with Kedro because workflows in Airflow are modelled and organised as [DAGs](https://en.wikipedia.org/wiki/Directed_acyclic_graph).
 
-## How to run a Kedro pipeline on Apache Airflow using a Kubernetes cluster
-
-The `kedro-airflow-k8s` plugin from GetInData | Part of Xebia enables you to run a Kedro pipeline on Airflow with a Kubernetes cluster. The plugin can be used together with `kedro-docker` to prepare a docker image for pipeline execution. At present, the plugin is available for versions of Kedro < 0.18 only.
-
-Consult the [GitHub repository for `kedro-airflow-k8s`](https://github.com/getindata/kedro-airflow-k8s) for further details, or take a look at the [documentation](https://kedro-airflow-k8s.readthedocs.io/).
-
-
 ## How to run a Kedro pipeline on Apache Airflow with Astronomer
 
-The following tutorial uses a different approach and shows how to deploy a Kedro project on [Apache Airflow](https://airflow.apache.org/) with [Astronomer](https://www.astronomer.io/).
+The following tutorial shows how to deploy an example [Spaceflights Kedro project](https://docs.kedro.org/en/stable/tutorial/spaceflights_tutorial.html) on [Apache Airflow](https://airflow.apache.org/) with [Astronomer](https://www.astronomer.io/).
```

Comment:
I still haven't done a thorough review, but after a quick scan I have one question:

Step 5 says "Visit the Airflow Webserver UI at its default address, http://localhost:8080", but my understanding was that the guide is based on Astronomer, "a managed Airflow platform" that additionally "provides a set of tools to help users get started with Airflow locally in the easiest way possible".

Given that we're not even registering an account on Astronomer for this tutorial, should we then clarify that we're using the Astro CLI https://github.com/astronomer/astro-cli rather than Astronomer then? Something like

```suggestion
The following tutorial shows how to deploy an example [Spaceflights Kedro project](https://docs.kedro.org/en/stable/tutorial/spaceflights_tutorial.html) on [Apache Airflow](https://airflow.apache.org/) with [Astro CLI](https://docs.astronomer.io/astro/cli/overview), a command-line tool created by [Astronomer](https://www.astronomer.io/) that streamlines the creation of local Airflow projects.
```

And possibly rename the file from `airflow_astronomer.md` to just `airflow.md`...

To note, `astro-cli` is Apache 2.0 + CommonsClause https://github.com/astronomer/astro-cli/blob/main/LICENSE hence this has implications for commercial projects. This is not something we necessarily have to write, but we should be aware of it.

Reply:
Thank you for review, @astrojuanlu. I agree with your points and would like to suggest an additional proposal. Perhaps instead of renaming we could conclude with additional section with instructions on how to migrate your local project to the remote Astro Cloud platform, which requires an Astronomer account. This guide looks like:
```
Log In to Astro Cloud
Create a New Deployment
Get Your Deployment's Cloud URL.
Deploy Your Project:
Once your local project is configured with the correct deployment ID, you can deploy your project to Astro Cloud using the Astro CLI. Navigate to your project’s root directory in the terminal, and then run:

astro deploy -d <deployment-id>
```

Reply:
Oh adding this at the end would be a nice touch. But I still would like to see `astronomer` gone from the URL. `kedro-airflow-k8s` is not Astronomer-specific, I believe. And we could potentially add info about other managed Airflow offerings in the future.

Reply:
Agree, I added Cloud deployment section and renamed file also